src/index.html

```diff
@@ -4,19 +4,115 @@
     <meta charset="UTF-8" />
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content="width=device-width, initial-scale=1.0"
     />
     <meta
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Moyo header</title>
+    <meta
+      http-equiv="Cache-Control"
+      content="no-cache, no-store, must-revalidate"
+    />
+    <meta
+      http-equiv="Pragma"
+      content="no-cache"
+    />
+    <meta
+      http-equiv="Expires"
+      content="0"
+    />
+    <title>Moyo Header</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="Moyo Logo"
+          class="logo_image"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="moyo_menu">
+          <li class="menu-item">
+            <a
+              href="#"
+              class="menu_link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="menu-item">
+            <a
+              href="#"
+              class="menu_link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="menu-item">
+            <a
+              href="#"
+              class="menu_link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="menu-item">
+            <a
+              data-qa="hover"
+              href="#"
+              class="menu_link"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="menu-item">
+            <a
+              href="#"
+              class="menu_link"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="menu-item">
+            <a
+              href="#"
+              class="menu_link"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="menu-item">
+            <a
+              href="#"
+              class="menu_link"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="menu-item">
+            <a
+              href="#"
+              class="menu_link"
+            >
+              Video
+            </a>
+          </li>
```

Comment:
Remember about correct indentation between parent and child elements. Each `<li>` and `<a>` should be indented with 2 spaces relative to its parent.